dependencies.gradle

```diff
@@ -3,6 +3,11 @@
 dependencies {
     implementation('com.github.GTNewHorizons:NotEnoughItems:2.4.12-GTNH:dev')
     implementation('curse.maven:cofh-core-69162:2388751')
+    
+    runtimeOnly('com.github.GTNewHorizons:Baubles:1.0.1.16:dev')
```

Comment:
It is better to use `runtimeOnlyNonPublishable` here

Reply:
All these four switches to `runtimeOnlyNonPublishable`?